index.html

```diff
@@ -148,29 +148,28 @@ <h2 class="pt-3" id="info">What <em>is</em> Standard?</h2>
-          drop out about twenty-one months later.
+          is a tournament format which consists of several recent <i>Magic: The Gathering</i> sets. Most sets enter
+          the format when they're released and drop out about twenty-one months later.<br>
+          The rotating group of sets generally grows until it contains eight. Then—with the next joining release—the oldest four sets
+          are dropped at once with the four most recent ones remaining. Hence the fresh one is added as fifth legal set before the pool
+          grows up to eight again. There are exceptions to this. It's confusing.
           </p>
           <p>
-          Generally Standard will grow until it would receive its ninth set, then instead it drops its oldest four sets
-          to receive it as a fifth set. There are exceptions to this. It's confusing.
-          </p>
-          <p>
-          A <b>Standard card</b> is a card printed or reprinted into a set currently in Standard.
+          A <b>Standard card</b> is a card in a set currently part of the legal pool.
           <a v-tippy class="tip"
-             title="e.g. As long as Naturalize has a reprint currently in Standard, all previous prints of Naturalize
-             are also considered Standard.">
+             title="Sets not only contain newly designed cards. Sometimes there are so called reprints of former cards as well.<br>
+                    e.g. As long as Naturalize has a reprint currently in Standard, all previous prints of Naturalize
+                    are also considered Standard legal.">
           </a>
           </p>
           <p>
-          A <b>Standard deck</b> contains only Standard cards (60+ of them), can optionally have a sideboard
+          A <b>Standard deck</b> contains 60+ Standard cards and can optionally have a sideboard
           <a v-tippy class="tip"
-            title='A sideboard is an optional second pool of cards past the main deck that cannot be used during games,
-            but whose cards can be swapped with those from the main deck between games. It is generally used to keep
-            counters to specific deck archetypes available without needing to classify any change as an official deck
-            edit.'>
-          </a> of up to 15 more Standard cards, and apart from basic lands does not have more than four copies of any
-          one card
+            title="A sideboard is a second pool of cards past the main deck that cannot be used during games,
+                  but those cards can be swapped with, or added to, the main deck inbetween games.
+                  Those extra cards are mostly situational and are used in competitive play to have answers for specific deck archetypes available.">
+          </a> of up to 15 additional such cards. Furthermore, apart from basic lands both does not have more than four copies of any
+          one card combined.
```

Comment:
I don't like words like "furthermore"; they take up so much room and do so little in return.
```suggestion
          </a> of up to 15 additional such cards. Apart from basic lands, the combined main deck and
          sideboard cannot have more than four copies of any card.
```